src/Reader/AbstractFeed.php

```diff
@@ -259,14 +265,15 @@ public function __call($method, $args)
      * Return an Extension object with the matching name (postfixed with _Feed)
      *
      * @param  string $name
-     * @return Extension\AbstractFeed
+     * @return null|Extension\AbstractFeed
```

Comment:
This change is a bit suspicious :thinking:

Reply:
It's what the code was already doing; just documenting it properly here.